tests/HUBComponentWrapperTests.m

```diff
@@ -63,6 +81,55 @@ - (void)testComponentStateRestoring
     XCTAssertEqualObjects([self.UIStateManager restoreUIStateForComponentModel:model], @"Groovy");
 }
 
+- (void)testHighlight
+{
+    HUBComponentMock * const component = [HUBComponentMock new];
+    id<HUBComponentModel> const model = [self componentModelWithIdentifier:@"model"];
+    HUBComponentWrapper * const componentWrapper = [self componentWrapperForComponent:component model:model];
+    UIView * const superview = [[UIView alloc] initWithFrame:CGRectZero];
+    [componentWrapper viewDidMoveToSuperview:superview];
+    
+    [self.gestureRecognizer touchesBegan:[NSSet setWithObject:[UITouch new]] withEvent:[UIEvent new]];
+    [componentWrapper handleGestureRecognizer:self.gestureRecognizer];
+    XCTAssertEqual(self.selectionStateFromWillUpdateDelegateMethod, HUBComponentSelectionStateHighlighted);
+    XCTAssertEqual(self.selectionStateFromDidUpdateDelegateMethod, HUBComponentSelectionStateNone);
+    
+    XCTestExpectation * const expectation = [self expectationWithDescription:@"Waiting for highlight"];
```

Comment:
What do you thing about using `(void)performAsynchronousTestWithDelay: block:` from `HUBViewControllerTests`? We could also extract utilities functions from `HUBViewControllerTests` and make them reusable.